bin/db_replicate

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+[[ "$TRACE" ]] && set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o noclobber
+
+START_TIME=$(date +%s)
+
+declare -A SERVICE_COUNTS
+
+if [ -z "$ENVIRONMENT" ]; then
```

Comment:
I think the services we stop/start are essentially any with active connections to the database and that these should be `backend-xi worker-xi backend-uk worker-uk`

Do you mind just setting that as an explicit environment variable here so we can just see what's getting stopped/started?

I know technically that requires a new docker image to be deployed but its unlikely to change anytime soon tbh